app/models/fs/message_draft.rb

```diff
@@ -36,24 +37,26 @@ def self.create_and_validate_with_fs_form(form_files: [], author:, fs_client: Fs
       fs_form_identifier = form_information&.dig('form_identifier')
 
       box = author.tenant.boxes.with_enabled_message_drafts_import.find_by("settings ->> 'dic' = ?", dic)
-      fs_form = Fs::Form.find_by(identifier: fs_form_identifier)
 
-      unless box && fs_form
-        messages << nil
+      if box.nil?
+        errors << form_file.original_filename
```

Comment:
poslime von komplet `failed_files << form_file` nadradena funkcia sa postara, ze ako to prezentovat.